projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -10,12 +15,15 @@ import {
 import { store as editorStore } from '@wordpress/editor';
 import { createInterpolateElement, useEffect, useState } from '@wordpress/element';
 import { __, _n, sprintf } from '@wordpress/i18n';
+import { get } from 'lodash';
```

Comment:
IIRC we're trying to avoid using Lodash, so probably better to use a different approach.

Reply:
This line is actually borrowed from another part of the code.
I'll make it simpler and remove the lodash get.